Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -0,0 +1,404 @@
+            'environments': environments, 'status': statuses, 'severity': severities,
+            'type': types
+        }}
+        body = remove_empty_elements(body)
+        response = self._http_request(method='POST', json_data=body, cookies=self._cookies,
+                                      url_suffix='api/v1/alerts')
+        return response
+
+    def update_alerts(self, alerts: List[str], status: Optional[str],
+                      closure_reason: Optional[str]) -> Dict:
+        """
+        Update the status of one or more alerts
+
+        Args:
+            alerts (list(str)): Reference IDs for the alert(s)
+            status (str): Desired status to update for the alert(s)
+            closure_reason (str): Reason for updating the alerts status to closed.
+
+        Returns:
+            response (Response): API response from Cyberint.
+        """
+        body = {'alert_ref_ids': alerts,
+                'data': {'status': status, 'closure_reason': closure_reason}}
+        body = remove_empty_elements(body)
+        response = self._http_request(method='PUT', json_data=body, cookies=self._cookies,
+                                      url_suffix='api/v1/alerts/status')
+        return response
+
+    def get_csv_file(self, alert_id: str, attachment_id: str,
+                     delimiter: bytes = b'\r\n') -> Iterable[str]:
+        """
+        Stream a CSV file attachment in order to extract data out of it.
+
+        Args:
+            alert_id (str): ID of the alert the CSV belongs to.
+            attachment_id (str): ID of the specific CSV file.
+            delimiter (str): Delimiter for the CSV file.
+
+        Returns:
+            row (generator(str)): Generator containing each line of the CSV.
+        """
+        url_suffix = f'api/v1/alerts/{alert_id}/attachments/{attachment_id}'
+        with closing(self._http_request(method='GET', url_suffix=url_suffix,
+                                        cookies=self._cookies, resp_type='all',
+                                        stream=True)) as r:
+            for line in r.iter_lines(delimiter=delimiter):
+                yield line.decode('utf-8').strip('"')
+
+
+def test_module(client: Client):
+    """
+    Test the connection to the API by sending a normal request.
+
+    Args:
+        client (Client): Cyberint API  client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    return_value = None
+    try:
+        result = client.list_alerts(*([None] * 10))
+        if result:
+            return_value = 'ok'
+    except DemistoException as exception:
+        if 'Invalid token or token expired' in str(exception):
+            return_value = 'Error verifying access token and / or environment, make sure the ' \
+                           'configuration parameters are correct.'
+        else:
+            return_value = str(exception)

```

Comment:
In case of exception, you return it as an output, why not raise an exception?

Reply:
I use the return_error function, and select one of the error messages to be displayed.

Reply:
You use `return_error` just in case you catch an exception that you have raised, but in this case, you return a value (not raising)